desktop-cosmic/cosmic-icons/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=cosmic-icons
+PKGSEC=COSMIC
+PKGDEP="pop-icon-theme"
+BUILDDEP="gtk-update-icon-cache icon-naming-utils just"
```

Comment:
`gtk-update-icon-cache` should be omitted - cache generation is triggered after installation and should not be run during build-time as it generates extraneous files.